packages/react/src/components/TimePickerSpinner/TimePickerSpinner.test.jsx

```diff
@@ -90,101 +97,199 @@ describe('TimePickerSpinner', () => {
   });
 
   it('work with strings', () => {
-    const wrapper = mount(<TimePickerSpinner {...timePickerProps} value="xyz" spinner />);
+    render(<TimePickerSpinner {...timePickerProps} labelText="Pick a time" value="xyz" spinner />);
 
-    wrapper.find('input').simulate('blur');
+    const input = screen.getByLabelText('Pick a time');
 
-    expect(wrapper.find('input').props().value).toEqual('');
+    expect(input).toHaveValue('xyz');
+    fireEvent.blur(input);
+    expect(input).toHaveValue('');
 
-    wrapper.find('.iot--time-picker__controls--btn.down-icon').first().simulate('click');
-    expect(wrapper.find('input').props().value).toEqual('23:00');
+    userEvent.click(screen.getByTitle('Decrement hours'));
+    expect(input).toHaveValue('23:00');
   });
 
   // NOTE: enzyme's `simulate` doesn't really simulate anything.
```

Comment:
Is this old comment still valid?

Reply:
nope. removed with the fancy suggest changes button. 🎉